test/controllers/concerns/parameters/user_test.rb

```diff
@@ -3,7 +3,12 @@
 class UserParametersTest < ActiveSupport::TestCase
   include Foreman::Controller::Parameters::User
 
-  let(:context) { Foreman::Controller::Parameters::User::Context.new(:api, 'user', 'update', editing_self) }
+  def setup
+    @auth_source_ldap = FactoryBot.create(:auth_source_ldap)
```

Comment:
This will create an `n` & user before every single test, which is slow (commits to the db). How about creating a `context` to test this specific feature or just creating the auth source where needed? 

Same for user & `let(:context)`.